webpack.config.babel.js

```diff
@@ -190,6 +190,9 @@ if (isProduction) {
       entry: 'docs',
       paths: staticSiteGeneratorConfig.paths,
       locals: staticSiteGeneratorConfig,
+      globals: {
+        Element: {},
```

Comment:
this is to allow the `PropTypes.instanceOf(Element)` rule, while building the docs `Element` is undefined and breaks the whole thing 😕